docs/tempo/website/traceql/_index.md

```diff
@@ -0,0 +1,251 @@
+
+```
+{ span.http.status_code >= 200 && span.http.status_code < 300 } | count() > 2
+```
+
+In this example, the search reduces traces to those spans where: 
+
+* `http.status_code` is in the range of `200` to `299` and 
+* the number of matching spans within a trace is greater than two. 
+
+Queries select sets of spans and filter them through a pipeline of aggregators and conditions. If a spanset is produced after evaluation on a trace, then this spanset (and by extension the trace) is included in the result set of the query.
+
+
+## Selecting spans
+
+TraceQL differentiates between two types of span data: intrinsics, which are fundamental to spans, and attributes, which are customizable key-value pairs. You can use intrinsics and attributes to build filters and select spans.
+
+In TraceQL, curly brackets `{}` always select a set of spans from the current trace. They are commonly paired with a condition to reduce the spans being passed in.
+
+This simple query will be evaluated on every span of every trace, one at a time.
+```
+{ .http.status = 200 }
+```
+
+If the trace being evaluated contains no spans with an attribute `http.status` with the value `200`, then no spans will be selected and this trace will not appear in the result set.
+
+If the trace does contain spans with an attribute `http.status` with the value `200`, then only those spans will be returned. The trace is reduced to only the set of spans that match the condition inside the `{}`. The result set will contain only this subset of spans matching the condition.
+
+### Intrinsic fields
+
+Intrinsic fields are fundamental to spans. These fields can be referenced when selecting spans. Note that custom attributes are prefixed with `.`, `span.` or `resource.` whereas intrinsics can be typed directly.
+
+
+| **Operation** | **Type** | **Definition**                        | **Example**            |
+|---------------|----------|---------------------------------------|------------------------|
+| status        | string   | status values are error, ok, or unset | { status = ok }        |
+| duration      | duration | end - start time of the span          | { duration > 100ms }   |
+| name          | string   | operation or span name                | { name = "HTTP POST" } |
+
+### Attribute fields
+
+Attribute fields are derived from the span and can be customized. Process and span attribute types are [defined by the attribute itself](https://github.com/open-telemetry/opentelemetry-proto/blob/b43e9b18b76abf3ee040164b55b9c355217151f3/opentelemetry/proto/common/v1/common.proto#L30-L38), whereas intrinsic fields have a built-in type. You can refer to dynamic attributes (also known as tags) on the span or the span's resource.
+
+Attributes in a query start with a scope (see Scoped attribute fields section below). To find traces with the `GET HTTP` method, your query could look like this:
+
+```
+{ .http.method = "GET" }
+```
+
+Or like this:
```

Comment:
I think we should remove this example. (Lines 84-> 87)

Reply:
Done.